src/securejoin/mod.rs

```diff
@@ -1022,8 +813,9 @@ async fn secure_connection_established(context: &Context, contact_chat_id: ChatI
     let msg = context
         .stock_string_repl_str(StockMessage::ContactVerified, addr)
         .await;
-    chat::add_info_msg(context, contact_chat_id, msg).await;
+    chat::add_info_msg(context, contact_chat_id, &msg).await;
     emit_event!(context, EventType::ChatModified(contact_chat_id));
+    info!(context, "{}", msg);
```

Comment:
No, this writes the stock string which is also done in the `could_not_establish_secure_connection` below.  I think it's a helpful note that the process is done.

Reply:
Stock strings are translated if you don't use english, so if users send us logs this will be in their local language. I'd rather output "Contact {} is verified" directly.